src/Development/IDE/Spans/Calculate.hs

```diff
@@ -157,40 +173,36 @@ getTypeLPat _ pat =
 
 getLHsType
     :: GhcMonad m
-    => TypecheckedModule
+    => [TypecheckedModule]
     -> LHsType GhcRn
-    -> m [(SpanSource, SrcSpan, Maybe Type)]
-getLHsType _ (L spn (HsTyVar U _ v)) = pure [(Named $ unLoc v, spn, Nothing)]
+    -> m [(SpanSource, SrcSpan, Maybe Type, SpanDoc)]
+getLHsType tms (L spn (HsTyVar U _ v)) = do
+  let n = unLoc v
+  docs <- getDocumentationTryGhc tms n
+  ty <- catchSrcErrors "completion" $ do
+          name' <- lookupName n
```

Comment:
`lookupName` doesn’t work on `ghc-lib`. See the CPP for completions to disable this.